src/polyswarm_api/__init__.py

```diff
@@ -994,16 +961,6 @@ def scan_files(self, files):
         """
         return self.loop.run_until_complete(self.ps_api.scan_files(files))
 
-    def scan_url(self, to_scan, name='url'):
```

Comment:
I'd prefer either keeping these single-file convenience functions. As is, this changeset makes the API very inconsistent, as hash search and file download both have singular convenience functions. It also introduces breaking changes that we may need to fixup in other code.

Reply:
Added the single artifact methods back, keeping compatibility.